add monetalis clydesdale to RWA collaterals (#274)
makerdao/governance-manual: collateral/monetalis-clydesdale.md

# MIP65 - Monetalis Clydesdale

MIP65 established a legal structure that borrows DAI from a vault, exchanges it for USDC in the PSM and then cashes out the USDC to USD. These funds are then invested in a mixture of short-term US bonds. The end result is US treasuries backing DAI.

The end assets are IB01 and IBTA ETFs.

> The DAI loan is collateralized 1:1 with US Treasuries, althought the true collateralization ratio may vary with the value of the underlying holdings.

Reviewer: ```suggestion
The DAI loan is collateralized 1:1 with US Treasuries, although the true collateralization ratio may vary with the value of the underlying holdings.
```

> On a quarterly basis, surplus treasuries should be sold and the excess returned to the Surplus Buffer, marking profits for MakerDAO and MKR holders.

Reviewer: ```suggestion
On a quarterly basis, surplus treasuries are sold and the excess returned to the Surplus Buffer, marking profits for MakerDAO and MKR holders.
```

Reviewer: Not sure if this is actually the case. But having 'this should happen' in the docs is not ideal.

Author: agree, the reason I went with should is because there is a chance that they aren't, since it's not controlled by smart contracts we can't enforce it with code. Happy to change though

> A Dune dashboard displaying the status of the deal may be viewed [here](https://dune.com/SebVentures/makerdao-mip65).

Reviewer: Also link to this? <https://forum.makerdao.com/t/mip65-monetalis-clydesdale-documentation-hq/17923>

>Page last reviewed: 2023-02-24  

> >Next review due: 2024-02-24  

Reviewer: I think I had the others of these as quarterly updates, given RWA stuff is changing semi-regularly.
